apps/login/src/lib/server/loginname.ts

```diff
@@ -35,6 +36,15 @@ export async function sendLoginname(command: SendLoginnameCommand) {
 
   const loginSettings = await getLoginSettings(command.organization);
 
+  const potentialUsers = users.result.filter((u) => {
+    const human = u.type.case === "human" ? u.type.value : undefined;
+    return loginSettings?.disableLoginWithEmail
+      ? human?.email?.isVerified && human?.email?.email !== command.loginName
+      : loginSettings?.disableLoginWithPhone
+        ? human?.phone?.isVerified && human?.phone?.phone !== command.loginName
```

Comment:
should this not rather match? like human?.email?.isVerified && human?.email?.email === command.loginName

`===` istead of `!==` ?

Reply:
The idea is that we exclude users from the query using the loginname query. if email is not allowed, we only allow users where email is not on the resource thus !==